unstructured/chunking/title.py

```diff
@@ -104,7 +104,7 @@ def chunk_by_title(
     # -- but a negative value is not
     if combine_text_under_n_chars < 0:
         raise ValueError(
-            f"'combine_text_under_n_chars' argument must be >= 0, got {combine_text_under_n_chars}"
```

Comment:
Trailing comma is not necessary here, is it? Why would we want that?

Reply:
These edits are coming up automatically when I try to commit--I believe it comes from the `[pre-commit](https://github.com/Unstructured-IO/unstructured#installation-instructions-for-local-development)` configuration.

This is what happens when I try to commit:
```
(other logging info above...)
ruff.....................................................................Failed
- hook id: ruff
- files were modified by this hook
```
Then I have unstaged changes (the added commas), so I add them and can then commit and push.

Reply:
Hmm, I see. The operation of that pre-commit hook would explain why it's not happening on my machine (and why CI didn't complain about it when I added those lines).

I think that hook is missing something ... here it is, `COM812` in the ignore line for `ruff`, you can see it in the `pyproject.toml` here, which is what CI uses https://github.com/Unstructured-IO/unstructured/blob/main/pyproject.toml#L21

That needs to be added on line 31 of `.pre-commit-config.yaml`, might need to uninstall and reinstall the pre-commit hook for that to take effect.

Reply:
🙏 Sounds good. I'll create s PR and do that.